terraform.go

```diff
@@ -100,10 +101,17 @@ func getTerraformEnv() []string {
 	env = append(env, "TF_LOG=") // so logging can't pollute our stderr output
 	env = append(env, "TF_INPUT=0")
 
+	// don't propagate the magic cookie
+	env = append(env, "TF_PLUGIN_MAGIC_COOKIE=")
+
 	if p := os.Getenv("TF_ACC_LOG_PATH"); p != "" {
 		env = append(env, "TF_LOG=TRACE")
 		env = append(env, "TF_LOG_PATH="+p)
 	}
+
+	for k, v := range wd.env {
```

Comment:
Do you need to skip any keys here, like magic cookie, etc?

Reply:
Would it be easier to just set the keys in a map and iterate once? Or are you trying to preserve override capability?

Reply:
Magic cookie gets skipped elsewhere. It would be easier to have the keys be a map, but I wanted to match the existing interface.